easy-ui-react/src/DataGrid/DataGrid.module.scss

```diff
@@ -0,0 +1,97 @@
+@use "../styles/common" as *;
+
+.DataGrid {
+  @include component-token(
+    "data-grid",
+    "header-bg",
+    design-token("color.blue.800")
+  );
+  @include component-token(
+    "data-grid",
+    "header-text",
+    design-token("color.white")
+  );
+  @include component-token("data-grid", "header-row-height", 48px);
+  @include component-token("data-grid", "body-row-height", 56px);
+  @include component-token("data-grid", "expand-btn-size", 24px);
+  @include component-token(
+    "data-grid",
+    "sort-icon-color",
+    design-token("color.gray.500")
+  );
+  @include component-token(
+    "data-grid",
+    "border-width",
+    design-token("shape.border_width.1")
+  );
+  @include component-token(
+    "data-grid",
+    "border-color",
+    design-token("color.gray.300")
+  );
+  @include component-token("data-grid", "sticky-shadow-size", 8px);
+  @include component-token("data-grid", "sticky-shadow-opacity", 0.5);
+  @include component-token(
+    "data-grid",
+    "sticky-shadow-color",
+    design-token("color.gray.500")
+  );
+  @include component-token("data-grid", "cell-bg", design-token("color.white"));
+  @include component-token(
+    "data-grid",
+    "cell-text",
+    design-token("color.blue.800")
+  );
+  @include component-token("data-grid", "cell-z-index", 1);
+  @include component-token("data-grid", "cell-stuck-z-index", 2);
+  @include component-token("data-grid", "column-z-index", 3);
+  @include component-token("data-grid", "column-stuck-z-index", 4);
+  @include component-token("data-grid", "sticky-left-offset", -16px);
+  @include component-token("data-grid", "sticky-right-offset", -12px);
+
+  border: component-token("data-grid", "border-width") solid
+    component-token("data-grid", "border-color");
+  border-radius: design-token("shape.border_radius.md");
+  /* stylelint-disable scss/operator-no-newline-after */
+  max-height: calc(
+    (component-token("data-grid", "border-width") * 2) +
+      component-token("data-grid", "header-row-height") +
+      (
+        component-token("data-grid", "body-row-height") *
+          component-token("data-grid", "max-rows")
+      )
+  );
```

Comment:
**question**: i think i get what this `max-height` is computing, but do we need it to prevent "height" overflow when expanding rows? or some other reason?

Reply:
yeah it's for setting an overflow on the table when a consumer sets `maxRows={}` to a certain number. that way if there are a lot of rows in the table the consumer can have it scroll past <x> number of rows instead of being super long. by default though this is disabled